web/app/js/components/Navigation.jsx

```diff
@@ -180,22 +218,37 @@ class NavigationBase extends React.Component {
     this.setState(state => ({ helpMenuOpen: !state.helpMenuOpen }));
   }
 
-  menuItem(path, title, icon) {
+  menuItem(path, title, icon, isCommunity) {
     const { classes, api } = this.props;
     let normalizedPath = this.props.location.pathname.replace(this.props.pathPrefix, "");
     let isCurrentPage = path => path === normalizedPath;
 
-    return (
-      <MenuItem
-        component={Link}
-        to={api.prefixLink(path)}
-        className={classes.navMenuItem}
-        selected={isCurrentPage(path)}>
-        <ListItemIcon>{icon}</ListItemIcon>
-        <ListItemText primary={title} />
-      </MenuItem>
-    );
+    if (isCommunity) {
```

Comment:
consider passing in `onClick`, that way you won't have to if/else based on `isCommunity`:
```js
menuItem(path, title, icon, onClick) {
...
    return (
      <MenuItem
        component={Link}
        onClick={onClick}
```

Reply:
Great tip, thanks - I didn't realize that React would skip the `onClick` attribute if no value is supplied to the `menuItem` function - solves the problem of the extra code block!